scripts/docker_run

```diff
@@ -13,15 +16,22 @@ docker build \
   --build-arg="USERNAME=$USERNAME" \
   --build-arg="UID=$(id -u)" \
   --build-arg="GID=$(id -g)" \
-  .
+  --quiet \
+  . > /dev/null
```

Comment:
Why do you need both --quiet and > /dev/null? 
Also, does this mean that we will silence all errors that can happen in the build?

Reply:
`> /dev/null` is sufficient 👍. 

Yes, it is silencing the build process and potential errors. It would be awesome to have the script return the container id as returned by `docker run --detach` though. This way other processes have a reference to it when they detach a container. They still could grep the container id using `docker ps --quiet --latest` though. 

https://github.com/project-oak/oak/pull/110#discussion_r294082171
https://github.com/project-oak/oak/pull/105#discussion_r293735671

Reply:
Balancing the two aspects it appears to me now that loosing the build log costs more than not getting the container id returned from the docker_run script. Working with `docker ps --quiet --latest` on the caller side to get the reference to the detached container.